tests/qchem/test_integrals.py

```diff
@@ -875,3 +874,492 @@ def test_gradient_repulsion(self, symbols, geometry, alpha, coeff):
 
         assert np.allclose(g_alpha, g_ref_alpha)
         assert np.allclose(g_coeff, g_ref_coeff)
+
+
+def generate_symbols_alpha_coeff():
```

Comment:
Why this is needed?

Reply:
Not needed. Just makes life easier so I don't have to rewrite ref data over and over inside the function bodies